internal/engine/live_update_build_and_deployer.go

```diff
@@ -139,7 +140,7 @@ func (lubad *LiveUpdateBuildAndDeployer) buildAndDeploy(ctx context.Context, cu
 	}
 
 	if len(toRemove) > 0 {
-		l.Infof("Will delete %d file(s) from container: %s", len(toRemove), cInfo.ContainerID.ShortStr())
+		l.Infof("Will delete %d file(s) from containe(r): %s", len(toRemove), cIDStr)
```

Comment:
is this supposed to read "container(s)"? or is the "r" silent? :eyes:

Reply:
🤐